Tasks/DockerV1/containerpush.ts

```diff
@@ -40,7 +40,10 @@ function dockerPush(connection: ContainerConnection, image: string, imageDigestF
 
 export function run(connection: ContainerConnection): any {
     let command = tl.getInput("command", true);
-    var commandArguments = tl.getInput("arguments", false); 
+    var commandArguments = tl.getInput("arguments", false);
+    if (commandArguments) {
+        commandArguments = commandArguments.replace(/\n/g, " ");
```

Comment:
Instead of duplicate the code in every command, should we have a common function to get arguments?

Reply:
Moved the code to dockerCommandUtils and added L0 tests for DockerV1.
Will take up DockerV2 L0s separately. (Created an internal tracking item).